src/include/migraphx/algorithm.hpp

```diff
@@ -76,6 +96,53 @@ void group_unique(Iterator start, Iterator last, Output out, Predicate pred)
     }
 }
 
+template <class Iterator, class Predicate, class Output>
+void group_find(Iterator start, Iterator last, Predicate pred, Output out)
```

Comment:
is `out` here a Callable object ?

Reply:
Yes just like the other group algorithms.